src/pages/instances/InstanceOverviewDevices.tsx

```diff
@@ -0,0 +1,67 @@
+import { FC } from "react";
+import { MainTable } from "@canonical/react-components";
+import { LxdInstance } from "types/instance";
+import {
+  LxdGPUDevice,
+  LxdNoneDevice,
+  LxdOtherDevice,
+  LxdProxyDevice,
+} from "types/device";
+
+interface Props {
+  instance: LxdInstance;
+}
+type InstanceOverviewDevices =
+  | LxdNoneDevice
+  | LxdGPUDevice
+  | LxdOtherDevice
+  | LxdProxyDevice;
```

Comment:
```suggestion
```
I don't think we need a new type here (see below). I think we should not use the same name for a component and a type.

Reply:
I understand not using the same name for the Component and type here, but removing the new types gives me the error that I was dealing with before. I cannot simply use the LxdDeviceValue type as we get another case of LxdISoDevice not having a name attribute, so I created a new type for all the devices but Iso and disks.

Reply:
I think with the current code, we don't need to access device.name, but we only rely on devicename. So this should be fine?

Reply:
Ahhh, I see. I understand where you are coming from now, thank you for clarifying.